decidim/decidim
# Fix admin language selector

#### :tophat: What? Why?
When having more than 5 languages set for the organizations, the language selector is changing from "tabs" to "select". It seems that in the latest develop branch when having the select language drop down and changing the language, the targeted language field is not displayed. 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #?
- Fixes #?

#### Testing
1. Create a new development application, and enable 5-6 languages
2. Login as admin, visit admin organization settings 
3. Go and select a new language for "Body for the admin terms of service*Required field"
4. See there is not possible
5. Apply patch 
6. Change language, and see everything is working as it should

### :camera: Screenshots
*Please add screenshots of the changes you are proposing*
![Description](URL)

:hearts: Thank you!
